README.md

```diff
@@ -1,8 +1,19 @@
 # styles
 The billinge group matplotlib style sheet.
 
-To use this style put the following into your plot generating code:
+## Workflow
+1. clone to local drive via `git clone https://github.com/pdfttp/styles.git`
+1. put the following into your plot generating code:
 ```python
 import matplotlib.pyplot as plt
 plt.style.use(os.path.join('/path/to/this/folder', 'billinge.mplstyle'))
 ```
+
```

Comment:
just to be explicit, please add "where the path points to the location of the repo on your local hard-drive"

Otherwise, this is great.